codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/config/ServiceConfigGenerator.kt

```diff
@@ -412,6 +415,27 @@ class ServiceConfigGenerator(
                 it.section(ServiceConfig.BuilderImpl)(this)
             }
 
+            if (runtimeMode.defaultToOrchestrator) {
+                rustTemplate(
+                    """
+                    /// Adds a runtime plugin to the config.
+                    ##[doc(hidden)]
+                    pub fn runtime_plugin(mut self, plugin: impl #{RuntimePlugin} + 'static) -> Self {
+                        self.runtime_plugins.push(#{SharedRuntimePlugin}::new(plugin));
```

Comment:
this can use `push_runtime_plugin`

Reply:
Fixed in cacf7bac9993f8785ba087aa185e4fb69d5c1419